velox/functions/sparksql/Arithmetic.h

```diff
@@ -35,15 +35,33 @@ struct RemainderFunction {
     if (UNLIKELY(n == 0)) {
       return false;
     }
-    // std::numeric_limits<int64_t>::min() % -1 could crash the program since
-    // abs(std::numeric_limits<int64_t>::min()) can not be represented in
-    // int64_t.
```

Comment:
Please move this comment instead of deleting.

Reply:
Fixed